source/about/subscription.rst

```diff
@@ -48,20 +73,15 @@ How to apply?
 ~~~~~~~~~~~~~
 
 To apply for the Mattermost Nonprofit License, `please complete this form <https://support.mattermost.com/hc/en-us/requests/new?ticket_form_id=17664739497236>`_.
-Please note that it may take up to six weeks for us to respond to your request.
+
+.. note::
+  
+  Please note that it may take up to six weeks for us to respond to your request.
 
 Mattermost educational license program
 ---------------------------------------
 
-For academic licensing, please see: https://mattermost.com/education/
-
-Mattermost Enterprise Cloud  
----------------------------
-
-Enterprises can inquire about Mattermost Cloud Enterprise, a single-tenant cloud-managed service for Mattermost Enterprise hosted by Mattermost, Inc. The system is offered on the same Kubernetes-based platform as the self-hosted edition, and managed by Mattermost, Inc. 
-
-For more information, talk to a `Mattermost Expert <https://mattermost.com/contact-sales/>`_
-
+For academic licensing, please visit us online: https://mattermost.com/education/
 
 Frequently asked questions
```

Comment:
@cwarnermm do we need to remove the credit card FAQ item now that we don't offer self-serve?

Reply:
Great catch, @nab-77! Will remove.